tools/c7n_org/c7n_org/cli.py

```diff
@@ -74,7 +74,10 @@
             'properties': {
                 'name': {'type': 'string'},
                 'email': {'type': 'string'},
-                'account_id': {'type': 'string'},
+                'account_id': {
+                    'type': 'string',
+                    'pattern': '^[0-9]{12}$',
```

Comment:
Its only for AWS?
It will not break GCP or azure?

Reply:
we use gcp project and azure subscription respectively for those in accounts.yml

Reply:
added some basic account config validation tests across providers with required values.